### PR TITLE
fix: SfAccordion folds after data updates

### LINKED ISSUE #1299: [BUG] SfAccordion folds after data updates
**Describe the bug**
SfAccordion item which is at first closed, folds after some interactions. 
E.g. SfAccordion content which includes some input and button, after interactions and updating the component, item folds and goes back to the default state (which is closed).

**How to reproduce**
1. Go to 'SfAccordion', let the Accordion be closed at first, include into content some input and button with interaction
2. Click on button to see interaction
4. See error, accordion goes back to the original state which is closed, but it should be open e.g. to see the provided data

### PR DESCRIPTION
**Pull request by justyna-13**
# Related issue
Closes #1299 

# Scope of work
Accordion keeps information about opened header in data 

# Screenshots of visual changes
No visual changes
# Checklist

### CODE REVIEW

File: packages/vue/src/components/organisms/SfAccordion/SfAccordion.vue [hunk 1]
```diff
@@ -70,18 +75,21 @@ export default {
...
+          child.isOpen = Array.isArray(this.openedHeader)
+            ? this.openedHeader.includes(child.header)
+            : this.openedHeader === child.header;
         });
       }
     },
     toggleHandler(slotId) {
-      if (!this.multiple && !Array.isArray(this.open)) {
+      if (!this.multiple && !Array.isArray(this.openedHeader)) {
         this.$children.forEach((child) => {
```
Comment (Michal-Dziedzinski) [inline packages/vue/src/components/organisms/SfAccordion/SfAccordion.vue:86]:
````suggestion
        this.$children.forEach(({_uid, isOpen, header}) => {

Comment (Michal-Dziedzinski) [reply packages/vue/src/components/organisms/SfAccordion/SfAccordion.vue:86]: And you can remove child below, wdyt?

File: packages/vue/src/components/organisms/SfAccordion/SfAccordion.vue [hunk 2]
```diff
@@ -49,6 +49,11 @@ export default {
       default: true,
     },
   },
+  data() {
+    return {
+      openedHeader: this.open,
```
Comment (Michal-Dziedzinski) [inline packages/vue/src/components/organisms/SfAccordion/SfAccordion.vue:54]: Isn't `openHeader` more accurate?

File: packages/vue/src/components/organisms/SfAccordion/SfAccordion.vue (hunk 1 above)
Comment (justyna-13) [reply packages/vue/src/components/organisms/SfAccordion/SfAccordion.vue:86]: sure, good idea :)

File: packages/vue/src/components/organisms/SfAccordion/SfAccordion.vue (hunk 2 above)
Comment (justyna-13) [reply packages/vue/src/components/organisms/SfAccordion/SfAccordion.vue:54]: yes, it's better :)

**Pull request merged**